### DISCUSSION TITLE
Setting to hide Application section in account page

### DISCUSSION THREAD

**Discussion by GioviQ** [keycloak/keycloak#32694] [Ideas]
![Immagine 2024-09-05 094503](https://github.com/user-attachments/assets/823f9895-3f30-464b-be4b-48f225834a79)

https://github.com/keycloak/keycloak/issues/32688#issuecomment-2331851307

I would like a settings to hide Application section in account page, because end user does not need / understand the purpose of this section.

**Comment by jonkoops** [comment]
There is no such setting, but you can create a custom theme that removes that section. Check out the [Learn to extend the Account Console](https://github.com/keycloak/keycloak-quickstarts/tree/latest/extension/extend-account-console#extend-account-console-learn-to-extend-the-account-console) quickstart, you'll have to [remove a line](https://github.com/keycloak/keycloak/blob/22f9d2077e6842e402a7db457964a8adf2c84c99/js/apps/account-ui/public/content.json#L15) from `content.json` in your theme.

**Comment by GioviQ** [comment]
yes that's what I expected, In fact I am studing https://docs.keycloakify.dev/ but it is not so easy.

Also in https://docs.keycloakify.dev/i18n it seems using other translations files so my PR translation experiment https://github.com/keycloak/keycloak/pull/32576 was useless.

Thanks

**Reply by jandrad** [reply]
For anyone looking for this, you can do it with keycloakify by claiming the `/account/account/assets/content.ts` using 

```
npx keycloakify own --path "account/assets/content.ts"
```

Then you can remove the MenuItems you don't need.

**Comment by martijndebruijn** [comment]
Workaround:
```
[data-testid="applications"] {
    display: none;
}
```
